kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -608,6 +624,23 @@ private void verifyBucketSpecified() throws ConfigException {
     }
   }
 
+  private void checkAutoCreateTables() {
+    Class<?> schemaRetriever = getClass(BigQuerySinkConfig.SCHEMA_RETRIEVER_CONFIG);
+
+    boolean autoCreateTables = getBoolean(TABLE_CREATE_CONFIG);
+    if (autoCreateTables && schemaRetriever == null) {
+      throw new ConfigException(
+              "Cannot specify automatic table creation without a schema retriever"
+      );
+    }
+
+    if (schemaRetriever == null) {
+      logger.warn(
+              "No schema retriever class provided; auto table creation is impossible"
```

Comment:
This log message seems a little confusing to me. As I read it, you would see this if you have `autoCreateTables` set to false. If so, I wouldn't expect to a message saying auto table creation is impossible--I didn't set auto create table in the config. I think removing this if check is probably a better choice.

Reply:
Make sense, removed this check!